ada-project-docs/wave_03.md

```diff
@@ -34,6 +34,12 @@ when I send a `PATCH` request to `/tasks/1/mark_complete`,
 
 then the task is updated, so that its `completed_at` value is the current date, and I get this response:
 
+`204 No Content`
+
+The response should have a mimetype of "application/json" to keep our API response type consistent.
+
+If after the `PATCH` request I fetch the task by sending a `GET` request to `/tasks/1`, I get this response:
```

Comment:
Wording is just a little confusing here. Maybe something like:

```suggestion
After I have made the 'PATCH' request, I can submit a 'GET' request to 'tasks/1, which will return the response:
```

Reply:
This reads much easier, thanks for the suggestion!